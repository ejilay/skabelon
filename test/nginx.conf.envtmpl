server {
        listen       {{ .PROXY_HTTP_PORT }};

        location /hls/ {
            proxy_pass http://{{ .API_HOST_PORT }}/api/v1/{{ .PROJECT }}/callback/hls_proxy/;
            proxy_intercept_errors on;
            error_page 301 302 307 = @handle_redirects;
         }

        location @handle_redirects {
           set $orig_loc $upstream_http_location;
           proxy_pass $orig_loc;
        }
}
