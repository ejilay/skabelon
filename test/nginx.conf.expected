server {
        listen       80;

        location /hls/ {
            proxy_pass http://api:8888/api/v1/restream/callback/hls_proxy/;
            proxy_intercept_errors on;
            error_page 301 302 307 = @handle_redirects;
         }

        location @handle_redirects {
           set $orig_loc $upstream_http_location;
           proxy_pass $orig_loc;
         }
}
